web/modules/custom/do_base/tests/src/Kernel/ExampleTest.php

```
@@ -0,0 +1,98 @@
+ *
+ * @package Drupal\do_base\Tests
+ */
+#[Group('DoBase')]
+class ExampleTest extends DoBaseKernelTestBase {
+
+  /**
+   * Tests addition.
+   */
+  #[DataProvider('dataProviderAdd')]
+  #[Group('addition')]
+  public function testAdd(int $a, int $b, int $expected, string|null $expectExceptionMessage = NULL): void {
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $a. Configured minimum length is 3. (undefined)

(ShortVariable)